topics/admin/metadata.yaml

```diff
@@ -35,6 +35,28 @@ material:
       - natefoo
       - bgruening
 
+  -
+    title: "Upgrading Galaxy"
+    type: "tutorial"
+    name: "upgrading"
+    zenodo_link: ""
+    hands_on: no
+    slides: yes
+    questions:
+      - What is the release cycle of Galaxy?
```

Comment:
`What do I need to prepare before I update my Galaxy?`?

Reply:
Added.